### PR TITLE
Adding basic documentation for MVC

### PR DESCRIPTION
**Pull request by hunterbarber**
## Fixes/Resolves:
Addition basic documentation for MVC

## Summary/Motivation:
No current documentation for already merged MVC model. This is only contains the basic info for documentation

## Changes proposed in this PR:
n/a

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/unit_models/mvc.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+^^^^^^^^^^^^^^^^^^^
+
+The condenser performance is related through the equations denoted by the footnote in the evaporator section.
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Complete condensation condition", ":math:`P_{out} >= P_{sat,out}\left(T_{out}\right)`"
+
+
```
Comment (adam-a-a) [inline docs/technical_reference/unit_models/mvc.rst:150]: Should we add links to the Class documentation for each of the units here?

**Review by adam-a-a — APPROVED** [review]
LGTM- I'd like this to get into the release, but we should probably make an issue to revisit and refine this documentation further, as needed.

File: docs/technical_reference/unit_models/mvc.rst (hunk 1 above)
Comment (hunterbarber) [reply docs/technical_reference/unit_models/mvc.rst:150]: Probably should, I agree. It was odd to document considering the modularity of the unit.

Comment (hunterbarber) [reply docs/technical_reference/unit_models/mvc.rst:150]: Addressed in [92df892](https://github.com/watertap-org/watertap/pull/963/commits/92df892b81e1519ca461994ace7ee28b4876d492)

File: docs/technical_reference/unit_models/mvc.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+Degrees of freedom for the MVC system are separated into the independent unit models. In this documentation, the
+description of the MVC system will only include the evaporator, compressor, and complete condenser unit models. The
+total system has 6 degrees of freedom in addition to the inlet feed state variables. Beginning from the feed, the
+evaporator has 3 degrees of freedom:
+
+   * 1 outlet condition (brine or vapor temperature or pressure)
+   * overall heat transfer coefficient
+   * area for heat exchange
+
+The condenser which the vapor enters has 3 degrees of freedom:
```
Comment (carson-tucker) [inline docs/technical_reference/unit_models/mvc.rst]: This should be compressor.

Comment (adam-a-a) [inline docs/technical_reference/unit_models/mvc.rst]:
```suggestion
The compressor which the vapor enters has 3 degrees of freedom:
```

**Review by carson-tucker — APPROVED** [review]

**Pull request merged**